### DISCUSSION TITLE
Cutting off song features

### DISCUSSION THREAD

**Discussion by Hauntinq** [jojo2357/Music-Discord-Rich-Presence#39] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/80594616/118414871-041d0c80-b69f-11eb-94c3-dcb76eba564b.png)
![image](https://user-images.githubusercontent.com/80594616/118414887-1bf49080-b69f-11eb-98e5-ff28a5741980.png)

**Comment by jojo2357** [comment, marked as answer]
The reason that it does this is because discord limits the rp to 32 characters so it has to cut off long artists and songnames in order to show at all. 

A small price to pay for salvation.

**Reply by Hauntinq** [reply]
Ohhhh ok lmao thnx for explaining

**Reply by jojo2357** [reply]
Np thanks for using and feel free to share ;)
